doc/source/overview_File_Dataset_Container.rst

```diff
@@ -38,9 +36,7 @@ A file is LOST if there are no known replicas of the file in Rucio, while at the
 
 * ``suppressed``: True/False
 
-This is a user settable flag. It indicates that the owner of the scope no longer needs the name to be present in the scope. Files that are suppressed (by default) do not show up in search and list operations on the scope. The setting of this flag is subject to conditions, e.g., one can not suppress a file while at the same time requesting it to be replicated somewhere.
-
-Note however that this flag will be ignored when explicitly listing contents of datasets/containers.
+This is a user settable flag. It indicates that the owner of the scope no longer needs the name to be present in the scope. Files that are suppressed (by default) do not show up in search and list operations on the scope. Note however that this flag will be ignored when explicitly listing contents of datasets/containers.
 
```

Comment:
suppressed is not really used in Rucio but more a external/worklow field no ?

Reply:
Yep. To be honest I wanted to remove it already from the page and just left it in as it was in the previous version. But I would be happy to remove it. It is not really significant.